app/Base/index.tsx

```diff
@@ -57,13 +57,17 @@ setMapboxToken(mapboxToken);
 if (sentryConfig) {
     init(sentryConfig);
 }
-
-ReactGA.initialize(trackingId, gaConfig);
-browserHistory.listen((location) => {
-    const page = location.pathname ?? window.location.pathname;
-    ReactGA.set({ page });
-    ReactGA.pageview(page);
```

Comment:
Do we not need to call ReactGA.pageview?

Reply:
done